lib/tasks/annotations.rake

```diff
@@ -0,0 +1,23 @@
+namespace :annotations do
+  desc 'Remediate annotations'
+  task remediate: :environment do
+    ActiveRecord::Base.transaction do
+      puts "Updating #{Annotot::Annotation.count} annotations"
+      Annotot::Annotation.find_each do |annotation|
+        print '.'
+        # Remove "@context"
+        data = JSON.parse(annotation.data)
+        data.delete('@context')
+
+        # Update uuid if a URI
+        if annotation.uuid.match?(%r{^http:\/\/.*})
```

Comment:
I assume there are no `https` URIs? Or should we check if the uuid isn't a uuid?

Reply:
Updated to now check for a uuid